src/transformers/pipelines/image_to_text.py

```diff
@@ -145,6 +145,12 @@ def preprocess(self, image, prompt=None):
         return model_inputs
 
     def _forward(self, model_inputs, generate_kwargs=None):
+        # Git model sets `model_inputs["input_ids"] = None` in `preprocess` (when `prompt=None`). In batch model, the
+        # pipeline will group them into a list of `None`, which fail `_forward`. Avoid this by checking it first.
+        if "input_ids" in model_inputs and type(model_inputs["input_ids"]) == list:
```

Comment:
```suggestion
        if "input_ids" in model_inputs and isinstance(model_inputs["input_ids"]), list):
```

No ?

Also I would put a single condition instead of 2 nested.